### PR TITLE
Update Main.php

### PR DESCRIPTION
**Pull request by PrimusLV**

### CODE REVIEW

**Comment by PrimusLV** [comment]
Why do you do Config::reload() ? You save it and in same moment load the same data.

**Comment by thebigsmileXD** [comment]
@PrimusLV why not. >:)

**Comment by PrimusLV** [comment]
negligible but waste of machine resources.

File: src/thebigsmileXD/VillagerTrade/Main.php [hunk 1]
```diff
@@ -37,12 +37,16 @@
...
+	
 	public $trader;
 	public $fakeChest = null;
 	public $inventory = null;
 	public $villager = null;
 	public $site = 0;
-
+	
+	/** @var Config */
+	public $profession;
```
Comment (thebigsmileXD) [inline src/thebigsmileXD/VillagerTrade/Main.php]: Why should this be here? Its unused and profeccsion is chosen based on the cloth a villager wears.

**Pull request closed without merging**